examples/xline-cluster-example.yml

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
```

Comment:
There are some duplicated sessions in this file. Can we use a template to generate this yaml file?

Reply:
There are indeed some duplicate declarations of PV here, but they are only needed for local testing at the moment. They will probably be removed when writing the example document, leaving only the final declaration of XlineCluster.

Reply:
Resolved in #15